ui/actions/nuxeo-workflow-button.js

```diff
@@ -52,6 +52,10 @@ import './nuxeo-action-button-styles.js';
           nuxeo-select {
             max-width: 380px;
           }
+          paper-button {
+            @apply --nx-button-primary;
+            @apply --nx-button-small;
```

Comment:
Do we really want to apply `small` and `primary` here?
This means that both buttons (`Start` and `Cancel`) would be smaller than all the other dialogs and also they will have the same style.

Reply:
old code.